src/mock/quick_p2p/network.rs

```diff
@@ -218,13 +216,27 @@ impl Queue {
         self.0.push_back(packet)
     }
 
-    fn pop(&mut self) -> Option<Packet> {
-        self.0.pop_front()
-    }
-
     fn is_empty(&self) -> bool {
         self.0.is_empty()
     }
+
+    // This function will pop random msg from the queue.
+    fn pop(&mut self, rng: &mut SeededRng) -> Option<Packet> {
```

Comment:
I would definitely rename it because to me `pop` suggests either `pop_front` or `pop_back`.
Maybe `pop_random_msg` ?

Reply:
As I understand, the `pop` function shall still be remained ?

Reply:
If you're asking me whether that's what I'm suggesting, it isn't. I'm just saying that I would rename it. If you think `pop` should remain for a different reason, can you explain it?

Reply:
I thought we may need a `sequenced pop` for some test.
However it turned not, and I have changed it.